AssociativeTable.cs

```diff
@@ -54,49 +48,40 @@ public void Put(TKey tempKey, TValue tempValue)
     /// Returns all values of Tuple elements inside Data array with passed key value.
     /// </summary>
     /// <param name="tempKey">The key to search for.</param>
-    public TValue Get(TKey tempKey)
+    public override TValue Get(TKey tempKey)
```

Comment:
lets change the name of parameters `tempKey` to just `key` , remember to update the docstring